parse_parameters_test.go

```diff
@@ -0,0 +1,148 @@
+package overflow
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hexops/autogold"
+	"github.com/onflow/cadence"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseArguments(t *testing.T) {
+	o, err := OverflowTesting()
+	require.NoError(t, err)
+	require.NotNil(t, o)
+
+	type Args struct {
+		inputArgs map[string]interface{}
+		code      string
+	}
+
+	type TestInput struct {
+		want    autogold.Value
+		want1   autogold.Value
+		wantErr autogold.Value
+		name    string
+		args    Args
+	}
+
+	tests := []TestInput{
+		{
+			name: "64bit integers",
+			args: Args{
+				code:      "u:UInt64, i:Int64",
```

Comment:
are we expected to handle variations of this wrt the spacing? e.g. `code: "u : UInt64, i: Int64"

Reply:
i am not sure if these tests are needed here at all anymore, since this logic now happends in underflow and they are already tested there. What do you think?

Reply:
yes this is not the place to test it as we've discussed the coverage is maintained by removing it and it's really an underflow test